OpenNefia.Content/UI/Element/UiWrapText.cs

```diff
@@ -101,5 +105,17 @@ private static string[] SplitString(string str, char[] splitChars)
                 startIndex = index + 1;
             }
         }
+
+        // function from https://stackoverflow.com/questions/17586/best-word-wrap-algorithm
+        private string[] SplitString(string str, PrototypeId<LanguagePrototype> lang)
+        {
+            switch (lang)
+            {
+                case var en when lang == LanguagePrototypeOf.English:
```

Comment:
Maybe make this into an `if` statement.

Reply:
makes sense to do it right now, but with potentially more languages this will arguably look cleaner